### ISSUE TITLE
Typo in FilePath.getPermissions's docstring

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6084]
|[<img alt="teratorn's avatar" src="https://avatars.githubusercontent.com/u/154069?s=50" width="50" height="50">](https://github.com/teratorn)| @teratorn reported|
|-|-|
|Trac ID|trac#6084|
|Type|defect|
|Created|2012-10-08 17:09:19Z|

"may not what is expected" should read "may not be what is expected"

Attachments:

* [twisted-python-filepath-FilePath-getPerissions.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/ecc/ecc8960598eece6384e562b381c74adbdf84c291/c4753c590698748fb67843dfd27c7bc6f884ce87.patch) (587 bytes) - added by Petit_Dejeuner on 2012-10-28 01:47:27Z - Fixes typo.
* [Isse6084TypoFilePath.getPermissionsDocstring.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/ecc/ecc8960598eece6384e562b381c74adbdf84c291/4c95956df6acdaa32e514f9d2915ceb9d331e855.patch) (566 bytes) - added by anitgen on 2012-11-16 23:24:36Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__6084 6084
type__defect defect
reporter__teratorn teratorn
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__easy easy
time__1349716159000000 1349716159000000
changetime__1353118934000000 1353118934000000
version__None None
owner__exarkun exarkun

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)<a name="note_1"></a>|@itamarst commented|
|-|-|

In order to get someone to look at this, you should add the "review" keyword when you submit a patch.

**Comment by twisted-trac** [comment]
|<img alt="kwidman's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|kwidman set owner to kwidman<br>kwidman set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="kwidman's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_3"></a>|kwidman removed owner<br>kwidman set status to `new`|
|-|-|

Simple patch; fixes the problem.

**Comment by twisted-trac** [comment]
|<img alt="abhikandoi2000's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|abhikandoi2000 set status to `closed`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun set status to `reopened`|
|-|-|

abhikandoi2000, please don't do that.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_6"></a>|@exarkun set owner to @exarkun<br>@exarkun set status to `new`|
|-|-|

Hi antigen.  Thanks for your interest in this ticket.  However, note that another developer has already contributed a fix for the issue.  The ticket is currently in the "review" phase, as described on ReviewProcess.  Further development work to fix the issue is not needed until the ticket has moved past the review phase.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_7"></a>|@exarkun commented|
|-|-|

Thanks for the patch (times two).  This change looks fine.  I'll apply it to trunk.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_8"></a>|@exarkun set status to `closed`|
|-|-|

(In [36418]) Correct a missing word in FilePath's API documentation

Author: Petit_Dejeuner
Reviewer: exarkun
Fixes: [#6084](https://github.com/twisted/twisted/issues/6084)
